editParagraph.php

```diff
@@ -23,18 +39,16 @@
 	<header class="container header-section">
 		<nav>
 			<img src="img/bars.svg" alt=""/>
-			<ul>
-				<li><a href="#">Log Out</a></li>
-			</ul>
 			<h1>Dashboard</h1>
 		</nav>
 	</header>
 	<main>
 		<section class="add-paragraph">
 			<h1>Edit Paragraph</h1>
-			<form class="add-form">
-				<textarea rows="20" cols="63" name="edit-paragraph">In my spare time I enjoy learning new creative skills which require me to use my hands and combine multiple ideas to form new creations. In recent years I have taught myself how to sew, knit and crochet. When I am not working or indulging in my hobbies you will find me providing support to young people in semi independence accommodation.
+			<form method="post" action="editParagraph.php" class="add-form">
+				<textarea rows="20" cols="63" name="edit-paragraph"> <?php echo $textToPopulate; ?>
```

Comment:
The textarea should be formatted in CSS, rather than the rows and columns specified here :( charlie told me off for it